src/main/java/core/basesyntax/handler/Balance.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.handler;
+
+import java.util.HashMap;
+
+public interface Balance {
+    void balance(String[] info, HashMap<String,Integer> storage);
+}

```

Comment:
it better to have one OperationHandler interface and 4 implementation classes of it